portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -848,6 +848,8 @@ public void addUserLocaleOptionsMessage(
 
 		PortalMessages.add(
 			httpServletRequest, PortalMessages.KEY_ANIMATION, false);
+		PortalMessages.add(
```

Comment:
I have real aversion and mistrust of anything that requires changing `PortalImpl`... specially for frontend. The less we touch here the better. Why is this even required?

Reply:
If you debug the receiving values from `cssClass`, you will notice the way that people can customize the alert just passing another style there. However, the user couldn't override the alert Icons like exclamation, full-circle-check for each Alert. This fix avoids having an info Alert with a danger symbol e.g.

<img width="892" alt="Screen Shot 2020-06-22 at 10 45 36" src="https://user-images.githubusercontent.com/7663513/85294851-842df880-b475-11ea-91a5-e71eaea75520.png">

Reply:
Hey @diegonvs, I think I understand your point, but could you please put this in an simple example so we can all understand?

If it is what you mean, I think I'd be rather inclined to ignore this for now. I don't think this is an API we want to create and promote unless totally necessary. Did you find a specific use case where you can use this?

Reply:
Because this part is responsible for rendering the user_locale_options JSP. https://github.com/liferay-frontend/liferay-portal/blob/e4e9201de2980247632bc2399b6649f9474c3c09/portal-impl/src/com/liferay/portal/util/PortalImpl.java#L855

Reply:
So what happens if you don't make this change? Anything?

How would you use this from other place? Have you seen a locale_option message triggered from somewhere else with a different type?

Reply:
I could don't touch on PortalImpl just matching the passed arguments to top_messages default values (and We're already doing this 08cfd05.

I'll remove my change on PortalImpl, since `displayType` is info by default.